### PR TITLE
ShellCheckBear.py: Importing CabalRequirement

### PR DESCRIPTION
**Pull request by naveentata**
CabalRequirement is imported to ShellCheckBear

Fixes https://github.com/coala/coala-bears/issues/1359

### CODE REVIEW

File: bears/shell/ShellCheckBear.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.CabalRequirement import (
```
Comment (Techievena) [inline bears/shell/ShellCheckBear.py]: Why the braces?

**Comment by Techievena** [comment]
Importing is not in imperative form. Also use `Closes` instead of `Fixes`

File: bears/shell/ShellCheckBear.py [hunk 2]
```diff
@@ -11,9 +15,14 @@ class ShellCheckBear:
 
     A gallery of bad code that can be detected is available at
     <https://github.com/koalaman/shellcheck/blob/master/README.md>.
+
+    Check <https://hackage.haskell.org/package/ShellCheck> for more
+    information.
     """
 
     LANGUAGES = {'sh', 'bash', 'ksh', 'dash'}
+    REQUIREMENTS = {CabalRequirement(package='shellcheck', version='0.4.4'),
```
Comment (jayvdb) [inline bears/shell/ShellCheckBear.py]: see `.ci/deps.shellcheck.sh` for the appropriate version.

**Comment by jayvdb** [comment]
ack a972fcc

**Comment by jayvdb** [comment]
needs rebase

**Comment by sils** [comment]
I object to this PR as per https://github.com/coala/coala-bears/issues/1359#issuecomment-277489451

**Comment by jayvdb** [comment]
blocked by sils

**Comment by jayvdb** [comment]
Since adding both is blocked by sils atm, could you only add the `DistributionRequirement` for now, as it is via apt-get that we are installing it currently (and .deb) and thus it is where the version is coming from.

Also in your commit message should describe where/why the version was chosen.

**Comment by naveentata** [comment]
This has been moved to https://github.com/coala/coala-bears/pull/1496

**Pull request closed without merging**